### ISSUE TITLE
Shorten visualisation chapter

### ISSUE THREAD

**Issue by lgatto** [carpentries-incubator/bioc-intro#71]
I would like to suggest to shorten the visualisation chapter by cutting the parts that we seem to skip systematically, namely

- the hexbin example
- all the ggplot customisation

and replace the latter with a link to relevant material (a ggplot vignette or chapter of the ggplot2 book) and a link to a ggplot2 figure gallery (there was a question asking about this at the Heidelberg workshop).

Comments, suggestions?

@csoneson , what do you think?

**Comment by csoneson** [comment]
I agree with removing the hexbin example. The customisation I'm less sure of - maybe it's useful to keep as an optional part in the end if nothing else to be able to point to the figures as an illustration that the appearance _can_ be adapted and modified (with a plot that is familiar to the learner)? I guess we also skipped the composition of multiple figures last time - maybe that's also a candidate for skipping/labeling as optional (it feels like more of a new concept than the customisation).
